src/viewmodels/vehicleViewModel.ts

```diff
@@ -37,10 +37,12 @@ export class VehicleViewModel
 	readonly _poweredAcceleration = store<number>(0);
 	readonly _poweredMaxSpeed = store<number>(0);
 	readonly _trackProgress = store<number>(0);
+	readonly _trackLocation = store<CarTrackLocation | null>(null);
 	readonly _spacing = store<number | null>(0);
 	readonly _x = store<number>(0);
 	readonly _y = store<number>(0);
 	readonly _z = store<number>(0);
+	readonly _xyz = compute(this._x, this._y, this._z, (x, y, z) => { return {x, y, z}; });
```

Comment:
I don't think it's necessary to introduce a new store for this, there's already the separate available stores and I prefer not to add memory and computing usage if it's not strictly necessary.